mitodl/odl-video-service
# Change prefix_id to a TextField

#### What are the relevant tickets?
Related to #787 

#### What's this PR do?
Changes `VideoS3.prefix_id` to a TextField so that more ElasticTranscoder prefix ids can be used.

#### How should this be manually tested?
- Set `ET_PRESET_IDS=1351620000001-200050,1351620000001-200040,1504127981769-6cnqhq,1504127981819-v44xlx,1504127981867-06dkm6,1504127981921-c2jlwt` in your .env file
- Upload some videos, they should be transcoded successfully.
